### ISSUE TITLE
About 'val_pickle_file' in 'train.py'

### ISSUE THREAD

**Issue by Jiangyu1181** [luting-hnu/EGRCNN#3]
Hi, thanks for your nice work.

However, I found a problem with your code. 

In 'train.py',  I find below:

"
train_pickle_file = '/home/luting/桌面/Bai/CD/dataset/train'
val_pickle_file = '/home/luting/桌面/Bai/CD/dataset/train'
"

In my opinion, you directly used the training set as the validation set and did not use the real validation set. I wonder if it is a clerical error?
